Functions/xASL_bids_CompareStructures.m

```diff
@@ -194,8 +198,7 @@ function printDifferencesAsWarnings(differences)
     % Iterate over differences
     if ~isempty(differences{1,1})
         for iT = 1:size(differences,1)
-            warning(differences{iT,1});
```

Comment:
This was for the getLogContent script support that you asked for. If you remove the warnings in `printDifferencesAsWarnings` and `printMissingAsWarnings` then you can't catch them later on...

Reply:
I see. Sorry. It is true that if I don't like them, then I can completely deactivate them :)

Can you please put the warnings back on? Also - we probably then want to run it from the testing script as printwarnings - true, but detailedoutput=false.

Reply:
> Can you please put the warnings back on? Also - we probably then want to run it from the testing script as printwarnings - true, but detailedoutput=false.

That was the idea :)